src/client/components/Input/Input.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Input.css';
+
+export default function Input({ classname, placeholder }) {
+  return <input className={classname} placeholder={placeholder} />;
+}
+
+Input.propTypes = {
+  placeholder: PropTypes.string.isRequired,
+  classname: PropTypes.string.isRequired,
```

Comment:
`className` shouldn't be required. Make it optional and provide default value in `defaultProps`.